tests/ImageSharp.Tests/Image/ImageFrameCollectionTests.NonGeneric.cs

```diff
@@ -90,7 +90,7 @@ public void AddNewFrame_Frame_FramesNotBeNull()
                         this.Collection.AddFrame(null);
                     });
 
-                Assert.StartsWith("Value cannot be null.", ex.Message);
+                Assert.StartsWith("Parameter \"source\" must be not null.", ex.Message);
```

Comment:
Could we also do ` Assert.Throws<ArgumentNullException>("source", () =>` here instead?

Reply:
I thought that was dealt with above?